topydo/Commands.py

```diff
@@ -32,7 +32,7 @@
     'depri': 'DepriCommand',
     'do': 'DoCommand',
     'edit': 'EditCommand',
-    'exit': 'ExitCommand', # used for the prompt
+    'exit': 'ExitCommand',  # used for the prompt
```

Comment:
Please revert the change in this line. I'd like to keep history as clean as possible.